src/query/server/query.go

```diff
@@ -484,6 +487,29 @@ func Run(runOpts RunOptions) RunResult {
 		if err != nil {
 			logger.Fatal("unable to setup downsampler for m3db backend", zap.Error(err))
 		}
+	case config.PromRemoteStorageType:
+		var opts promremote.Options
+		opts, err = promremote.NewOptions(cfg.PrometheusRemoteBackend, scope)
```

Comment:
nit: This could just be `opts, err := promremote.NewOptions(cfg.PrometheusRemoteBackend, scope)` yeah? Given shadowing err again would be just fine and opts is declared only just above here.